WookieeMatt/Essence20
# New interaction between Resources/Powers/Weapons

In this change
- Implements https://github.com/WookieeMatt/Essence20/issues/171
- Resource item renamed to Class Feature
- Added a dropdown next to Power and Weapon name labels for selecting a Class Feature
- Rolling a Weapon or Power that has a Class Feature will decrement it by one
- Included the Class Feature in the Weapon and Power rolls
- Personal Power is a Class Feature that is created by default when a new PR actor is created

Testing
- This includes a template change, so restart Foundry. If it won't load, you may need to delete your existing Resources, Powers, and Weapons.
- Create a new PR. It should have a Class Feature called Personal Power. Make the current value some non-zero number.
- Create a weapon on the PR and select Personal Power from the dropdown. Rolling the Weapon should act as normal, but also decrement Personal Power and include it in the chat message.
- You can do the same with a Power, and make it use your own Class Feature
